geomesa-convert/geomesa-convert-json/src/main/scala/org/locationtech/geomesa/convert/json/JsonFunctionFactory.scala

```diff
@@ -22,3 +23,36 @@ class JsonFunctionFactory extends TransformerFunctionFactory {
 
   override val functions: Seq[TransformerFn] = Seq(json2string)
 }
+
+class JsonMapListFunctionFactory extends TransformerFunctionFactory with MapListParsing {
+  override def functions = Seq(jsonListParser)
+
+  import scala.collection.JavaConverters._
+
+  def getPrimitive(p: JsonPrimitive) = {
+    if (p.isBoolean) {
+      p.getAsBoolean
+    } else if (p.isNumber) {
+      p.getAsString
```

Comment:
do you want numbers to be strings? that doesn't match the other json parsing does it?